src/Microsoft.Extensions.WebEncoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -17,14 +20,23 @@ public static class EncoderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/>.</param>
         /// <returns>The <see cref="IServiceCollection"/> instance after the encoders have been added.</returns>
-        public static IServiceCollection AddWebEncoders(this IServiceCollection services)
+        public static void AddWebEncoders(this IServiceCollection services)
```

Comment:
It would be really hard to enforce, and we don't want to do that. The reason to not return the IServiceCollection is to clearly differentiate the cases where you have a builder you can use to further configure services from the ones that don't.